lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -114,6 +113,89 @@ def define_let_it_be_methods(identifier, **modifiers)
 
       let(identifier, &let_accessor)
     end
+
+    def define_freezing_hooks
+      # Prevent hooks from being defined several times
+      return if instance_variable_get(:"#{PREFIX}hooks_defined")
+
+      before(:all) do
+        let_it_be_objects = instance_variable_get(:"#{TestProf::LetItBe::PREFIX}let_it_be_objects")
+
+        let_it_be_objects.each { |object| Freezer.deep_freeze(object) }
+      end
+
+      prepend_after do |example|
```

Comment:
Can we make this hook global? (`config.after`) Probably, with checking for the presence of `let_it_be_objects` inside it.

Reply:
Good idea, I'll give it a shot. At a glance it seems like both hooks can be made global.